pgmpy/models/BayesianModel.py

```diff
@@ -373,11 +373,13 @@ def dfs(node):
             return descendents
 
         from pgmpy.independencies import Independencies
```

Comment:
@khalibartan I also found some local import statements. I have not removed them. Maye be I'll send a different PR to remove all local import statements.